test/views/AboutSimilarGoodsViewSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import forms.AboutSimilarGoodsFormProvider
+import models.{DraftId, NormalMode}
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.AboutSimilarGoodsView
+
+class AboutSimilarGoodsViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "aboutSimilarGoods"
+
+  val form: AboutSimilarGoodsFormProvider = app.injector.instanceOf[AboutSimilarGoodsFormProvider]
+
+  val view: AboutSimilarGoodsView = app.injector.instanceOf[AboutSimilarGoodsView]
+
+  val viewViaApply: HtmlFormat.Appendable  = view(form.apply(), NormalMode, DraftId(1L))(fakeRequest, messages)
```

Comment:
Using this as a template for other view tests. Can all the viewViaApply in tests explicitly state the apply method which is done on the form as it gives clarity and there are `view.render` and `view.f` i.e. `view.apply(form.apply(), NormalMode, DraftId(1L))(fakeRequest, messages)`, thanks.